libsolidity/lsp/FileRepository.cpp

```diff
@@ -34,9 +34,25 @@ using namespace solidity::frontend;
 
 using solidity::util::readFileAsString;
 using solidity::util::joinHumanReadable;
+using solidity::util::Result;
 
-FileRepository::FileRepository(boost::filesystem::path _basePath): m_basePath(std::move(_basePath))
+FileRepository::FileRepository(boost::filesystem::path _basePath, std::vector<boost::filesystem::path> _includePaths):
+	m_basePath(std::move(_basePath))
 {
+	setIncludePaths(move(_includePaths));
+}
+
+void FileRepository::setIncludePaths(std::vector<boost::filesystem::path> _paths)
```

Comment:
This is not a simple setter. It makes some opinionated decisions. I think we should rename it to make it very clear:.
```suggestion
void FileRepository::setUpIncludePaths(std::vector<boost::filesystem::path> _requestedPaths)
```

Reply:
What about this one? Looks like it got lost among other comments.